### PR TITLE
WIP jsonalchemy: clean up of MARC fields definition

### PR DESCRIPTION
**Pull request by egabancho**
See #1722, #2075, #2348, #2651

### CODE REVIEW

File: contrib/marc/recordext/fields/01X_09X.cfg [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        marc, '909CO', {'group':value['i'], 'number':value['s'], 'id':value['o'], 'set':value['p'], 'previous_set':value['q']}
+    producer:
+        json_for_marc(), {'0248_a': 'id', '0248_p': 'set', '0248_q': 'previous_set'}
+
+@extend
+standard_identifier:
+    creator:
+        @legacy((("024", "0247_", "0247_%"), ""),
+                ("0247_a", "value"),
+                ("0247_2”, "schema"))
```
Comment (jirikuncar) [inline contrib/marc/recordext/fields/01X_09X.cfg]: `s/”/"/`

File: contrib/marc/recordext/fields/01X_09X.cfg [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        @legacy((("022", "022__", "022__%"), ""),
+                ("022__a", "issn", ""))
+        marc, '022..', {'issn': value['a'], 'issn-l': value['l'], 'incorrect': value['y']}
+    producer:
+        json_for_marc('022[_01]_'), {'a': 'issn', 'l': 'issn-l', 'y': 'incorrect'}
+        json_for_marc('022[^_01][^_]'), {'022__a': 'issn', '022__l': 'issn-l', '022__y': 'incorrect'}
+
+@extend
+oai:
+    """The second rule for marc is deprecated.
```
Comment (jirikuncar) [inline contrib/marc/recordext/fields/01X_09X.cfg]: Can we go for standard PEP257 docstring formatting?

File: contrib/marc/recordext/fields/5XX.cfg [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+abstract_secondary:
+    creator:
+        @legacy((("590", "590__", "590__%"), ""),
+                ("590__a", "summary"),
+                ("590__b", "expansion")
+                ("590__y", "language"))
+
+        marc, "590..", {'summary':value['a'], 'expansion':value['b'], 'language':value['y'] }
+    producer:
+        json_for_marc(), {"590__a": "sumary", "590__b": "expansion" 590__y": "language"}
```
Comment (jirikuncar) [inline contrib/marc/recordext/fields/5XX.cfg]: `"expansion" 590__y": "language"}` -> `"expansion",  "590__y": "language"}`

**Comment by jirikuncar** [comment]
@egabancho can you please make consistent usage of `'` and `"` in field definition?

**Comment by jirikuncar** [comment]
Please move `contrib` inside *Invenio* package (`invenio/contrib`).

**Comment by egabancho** [comment]
Unnecessary, see https://github.com/inveniosoftware/jsonalchemy

Only a double check is needed to verify that all the fields and subfields included in this PR are represented.

**Pull request closed without merging**